ContextFeatures/ContextRelativePoses.h

```diff
@@ -29,6 +29,23 @@
 #include "BoosterInputData.h"
 #include "IntegralImage.h"  // for BoxPosition
 
+namespace isnannamespace {
```

Comment:
This namespace had to be added because if anybody declares isnan, we have a redeclaration here (which was my case with Qt adding mathcalls.h)